frontend/src/metabase/css/core/arrow.module.css

```diff
@@ -2,13 +2,13 @@
 
 /* TODO: other arrow directions */
 
-:global(.arrow-right) {
+.arrowRight {
```

Comment:
> There is only one place that this class is used, it might make more sense to put it into a local `.modules.css` file that's specific to the component. Let me know what you think.

Yeah, I think so, so we wouldn't expect to use this elsewhere. If this file is in `css/core` it feels as if it's meant to be used through out the app.